outputs/aws.go

```diff
@@ -189,3 +192,76 @@ func (c *Client) PublishTopic(falcopayload types.FalcoPayload) {
 	c.Stats.AWSSNS.Add(OK, 1)
 	c.PromStats.Outputs.With(map[string]string{"destination": "awssns", "status": OK}).Inc()
 }
+
+// SendCloudWatchLog sends a message to CloudWatch Log
+func (c *Client) SendCloudWatchLog(falcopayload types.FalcoPayload) {
+	svc := cloudwatchlogs.New(c.AWSSession)
+
+	f, _ := json.Marshal(falcopayload)
+
+	c.Stats.AWSCloudWatchLogs.Add(Total, 1)
+
+	if c.Config.AWS.CloudWatchLogs.LogStream == "" {
+		streamName := fmt.Sprintf("falcosidekick-%d", time.Now().UTC().UnixNano())
```

Comment:
I think there's a bat pattern here. If `LogStream` value is empty in configuration, it means `falcosidekick` will create a new stream at each restart.

I think we should use a static name for the stream if the value is empty, and check only at bootstrap if it exists or not. The creation will be done at this stage too (with a creation in logs, as you do).

It also matters to add an IAM Policy example for allowing `falcosidekick` to create stream/push events (we should do it with SQS, SNS, Lambda too, btw)

Reply:
done @Issif PTAL